src/openassetio-core/include/openassetio/hostApi/Manager.hpp

```diff
@@ -1481,6 +1484,261 @@ class OPENASSETIO_CORE_EXPORT Manager final {
+   * as an exception, either from the @ref manager plugin (for errors
+   * not specific to the entity reference) or as a
+   * @fqref{errors.BatchElementException}
+   * "BatchElementException"-derived error.
+   *
+   * @param entityReference An @ref entity_reference to query
+   * the specified relationship for.
+   *
+   * @param relationshipTraitsData The traits of the relationship to
+   * query.
+   *
+   * @param pageSize The size of each page of data. The page size is
+   * fixed for the lifetime of pager object given to the @p
+   * successCallback. Must be greater than zero.
+   *
+   * @param relationsAccess The intended usage of the returned
+   * references.
+   *
+   * @param context The calling context.
+   *
+   * @param resultTraitSet A hint as to what traits the returned
+   * entities should have.
+   *
+   * @param errorPolicyTag  Parameter for selecting the appropriate
+   * overload (tagged dispatch idiom). See @ref
+   * BatchElementErrorPolicyTag::Exception.
+   *
+   * @return @ref EntityReferencePager pointer. Pages over
+   * an unbounded set of entityReferences related to the input
+   * entityReference.
+   *
+   * @throws errors.InputValidationException if @p pageSize is zero.
+   *
+   * @throws errors.NotImplementedException Thrown when this method is
+   * not implemented by the manager. Check that this method is
+   * implemented before use by calling @ref hasCapability with @ref
+   * Capability.kRelationshipQueries.
+   *
+   * @see @ref Capability.kRelationshipQueries
+   */
+  EntityReferencePagerPtr getWithRelationship(
+      const EntityReference& entityReference, const trait::TraitsDataPtr& relationshipTraitsData,
+      size_t pageSize, access::RelationsAccess relationsAccess, const ContextConstPtr& context,
+      const trait::TraitSet& resultTraitSet,
+      const BatchElementErrorPolicyTag::Exception& errorPolicyTag = {});
+
+  /**
+   * Query an entity reference that are related to the input
+   * reference by the relationship defined by a set of traits and
+   * their properties.
+   *
+   * See documentation for the <!--
+   * --> @ref getWithRelationship(const EntityReferences&, <!--
+   * --> const trait::TraitsDataPtr&, size_t, <!--
+   * --> access::RelationsAccess, const ContextConstPtr&, <!--
+   * --> const RelationshipQuerySuccessCallback&, <!--
+   * --> const BatchElementErrorCallback& errorCallback, <!--
+   * --> const trait::TraitSet&)
+   * "callback variation" for more details on resolution behaviour.
+   *
+   * For successful references, the corresponding element of the result
+   * is populated with the available data for the requested set of
+   * traits.
+   *
+   * Otherwise, the corresponding element of the result is populated
+   * with an error object detailing the reason for the failure to
+   * fetch the related entities for that particular relationship.
+   *
+   * Errors that are not specific to an entity will be thrown as an
+   * exception, failing the whole batch.
```

Comment:
https://github.com/OpenAssetIO/OpenAssetIO/pull/1308/commits/319407de054c9146009d56241f77f7b8d4492d5b

Reply:
I thought these chunks for exception/variant behaviour were identical ... and _obviously_ I didn't bother to read them. https://github.com/OpenAssetIO/OpenAssetIO/pull/1308/commits/a5c0ca9e8d3ea58dc4b2651b66ac7fb0c7d4a0de

Reply:
> pages over an unbounded set of entity references related to the input entity reference, or an error.

I think the `, or an error` is a copy-paste error?  Otherwise, what does it mean?

Reply:
I think it's just an actual writing error. Fixed this coincidentally as part of another comment, glad I spotted this one is still open. : https://github.com/OpenAssetIO/OpenAssetIO/pull/1308/commits/78bbc6da3c89788a389dc92e02ae7e2d2c34865a